.storybook/components/SliderStory.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Slider from '../../components/Slider';
+import TextInput from '../../components/TextInput';
+
+storiesOf('Slider', module)
+  .addWithInfo(
+    'default',
+    `
+      lorem
```

Comment:
Add a real description here and anything else that users may need to know about this React component. [Here's our description from the website](http://carbondesignsystem.com/components/slider/code):

> Sliders provide a visual indication of adjustable content, where the user can move the handle along a horizontal track to increase or decrease the value.